sdk/java-sdk/src/main/java/kalix/javasdk/eventsourcedentity/EventSourcedEntityOptions.java

```diff
@@ -39,8 +39,12 @@ public interface EventSourcedEntityOptions extends EntityOptions {
    */
   EventSourcedEntityOptions withSnapshotEvery(int numberOfEvents);
 
+  @Override
   EventSourcedEntityOptions withPassivationStrategy(PassivationStrategy strategy);
```

Comment:
This option is ignored, should I make it deprecated in java and scala sdks?

Reply:
yep, we should deprecate them.

Reply:
added, the thing is that I can't add this on an EntityOptions class, because the passivation strategy is not ignored for replicated entities, only for VE and ES.

Reply:
That's annoying. 

Can we do the following...
We move this method to each of the variants of `EntityOptions` and we deprecate it only in ES and VE. True, someone depending on `EntityOptions` directly may be impacted, but that's unlike to happen, IMO. 

Maybe better to do it in another PR in that case.

Reply:
Wait, that's exactly what you did. I don't see the issue then.

Reply:
Yeah, it was only a comment about why I did it this way.

Reply:
What does passivation mean for replicated/crdt entities? Do we have some special GC:y cleanup for them somehow?

Reply:
it is used in `context.setReceiveTimeout(serviceSettings.passivationTimeout.duration)` and
```
    case ReceiveTimeout =>
      context.parent ! ReplicatedEntityManager.Passivate
```

Reply:
I see, looking into how they are implemented I definitely did not have the whole picture.